### PR TITLE
Storing all variations of relIso and miniIso with different effective areas

### PR DESCRIPTION
**Pull request by tomcornelis**
Verified: no impact on current implementation of lepton mva's (see tests)

### CODE REVIEW

File: multilep/test/testing/tests.log [hunk 1]
```diff
@@ -1,42 +1,57 @@
...
+      _relIso_80X                                        mean:   0.0761 -->   0.0697     rms:   0.1926 -->   0.1861    
 
 --------------------------------------------------------------------------------------------------
 
 Running test: Summer16MiniAODv3 --> OK
-   Removed: _miniIsoCharged_80X,_miniIsoCharged_Spring15
+   Changed:
+      _leptonMvaTOP                                      mean:   0.8204 -->   0.8233     rms:   0.4736 -->   0.4705    
+      _leptonMvaTTH                                      mean:   0.8582 -->   0.8596     rms:   0.4314 -->   0.4301    
+      _leptonMvatZq                                      mean:   0.8088 -->   0.8139     rms:   0.4806 -->   0.4742
```
Comment (tomcornelis) [inline multilep/test/testing/tests.log]: This is restoring the original values, which were broken in a former test run included in this pull request (https://github.com/GhentAnalysis/heavyNeutrino/commit/175de672638f563a8de6f912ddca8f7c92f4619f#diff-999fefd3e11bd7f25f9e57046ff7397dR53-R55).

File: multilep/test/testing/tests.log [hunk 2]
```diff
@@ -1,42 +1,57 @@
...
-    Effective areas have of course no effect on the charged component
+    Small fix
 
 --------------------------------------------------------------------------------------------------
 
 Running test: Run2018-17Sep2018 --> OK
-   Removed: _miniIsoCharged_80X,_miniIsoCharged_Spring15
+   Changed:
+      _miniIso_80X                                       mean:   0.2014 -->   0.1991     rms:   0.4842 -->   0.4833    
+      _relIso_80X                                        mean:   0.2742 -->   0.2681     rms:   0.5991 -->   0.5962
```
Comment (tomcornelis) [inline multilep/test/testing/tests.log]: In the earlier commit, earlier in this pull request, I accidentally put again the 94X effective areas also  in these 80X variables. Fixed now.

**Comment by tomcornelis** [comment]
@kskovpen All variations of relIso and miniIso now stored in the tree, without impact on existing leptonMva's.

**Comment by kskovpen** [comment]
Thanks a lot, @tomcornelis ! Wonderful.

**Pull request merged**